README.md

```diff
@@ -1,53 +1,134 @@
 [![Automatic version updates](https://github.com/ZOSOpenTools/v8port/actions/workflows/bump.yml/badge.svg)](https://github.com/ZOSOpenTools/v8port/actions/workflows/bump.yml)
 
-v8
+# v8
 
 A port of Google V8 JavaScript engine to z/OS Open Tools project
 
-# Applying the patches
+## Workflow 
+
+### Applying the patches
 Since the [cipd executable](https://chromium.googlesource.com/infra/luci/luci-go/+/master/cipd/) is not yet ported on z/OS, the following steps are
 currently used to update V8 and its dependencies, and apply the z/OS port, after
 the initial [fetch v8](https://v8.dev/docs/source-code) (also included below with a comment).
 
+This workflow requires two platforms.  One a z/OS® UNIX System Services (z/OS UNIX) which is 
+the target platform.  The second is a generic platform used to get files normally acquired via `cipd`.
+
+
+#### Linux plaform (non z/OS® UNIX System Services (z/OS UNIX)) setup
+
+This is the workflow on the non-z/OS UNIX platform for obtaining files via `cipd`.  
+There are two aspects to this workflow.  One which is done once and one which
+is done everytime you want to sync.
+
+This workflow is done once.
+
 ```
-# Do this only once:
-mkdir ~/v8base
-cd v8base
-fetch v8
-cd v8
+$ cd $HOME
+$ git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+$ export PATH=~/depot_tools:$PATH
+$ fetch v8
+$ cd $HOME/v8
+$ git checkout main
+```
+This workflow is done afterwards and everytime you want to sync.
 
-# Do this to update V8 and its dependencies. The reset command is required
-# as the current patches are valid as of the specified commit.
-git pull
-git reset fce38915e4c7c73c0e7f77bb13e2a78514968d35 --hard
-gclient sync
-rm -rf test tools third_party
 ```
-From another platform (e.g. Linux), perform the same steps above (but without
-the rm command), then transfer the directories test, tools and third_party to
-your z/OS machine.
+$ cd $HOME/v8
```

Comment:
Add (as originally shown):
```
$ git pull
```
before the git reset, and add:
```
$ gclient sync
```
after the git reset.

Reply:
I'm confused by this comment.  Here is the workflow on the as shown on the linux box:


#### Linux plaform (non z/OS® UNIX System Services (z/OS UNIX)) setup

This is the workflow on the non-z/OS UNIX platform for obtaining files via `cipd`.
There are two aspects to this workflow.  One which is done once and one which
is done everytime you want to sync.

This workflow is done once.

```
$ cd $HOME
$ git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
$ export PATH=~/depot_tools:$PATH
$ fetch v8
$ cd $HOME/v8
$ git checkout main
```
This workflow is done afterwards and everytime you want to sync.

```
$ cd $HOME/v8
$ git reset fce38915e4c7c73c0e7f77bb13e2a78514968d35 --hard
$ tar -cvf x.tar test tools third_party
```

Where in that sequence is the git pull and gclient sync done?